monkey/infection_monkey/network/relay/relay_connection_handler.py

```diff
@@ -34,7 +34,18 @@ def handle_new_connection(self, sock: socket.socket):
             self._relay_user_handler.disconnect_user(addr)
         else:
             try:
-                self._pipe_spawner.spawn_pipe(sock)
+                self._pipe_spawner.spawn_pipe(sock, self._handle_pipe_data)
                 self._relay_user_handler.add_relay_user(addr)
             except OSError as err:
                 logger.debug(f"Failed to spawn pipe: {err}")
+
+    def _handle_pipe_data(self, sock: socket.socket, _: bytes):
+        """
+        Handle data received on a pipe.
+
+        :param sock: The socket the data was received on.
+        :param data: The data received.
```

Comment:
There's no `data` param.

Reply:
Also it doesn't make sense for this to be called `_handle_pipe_data` as it is not doing that.

Reply:
maybe "_on_pipe_received_data"?